examples/multi_invokee_rendering/source/multi_invokee_rendering.cpp

```diff
@@ -0,0 +1,200 @@
+			if (imguiManager != nullptr) {
+				imguiManager->add_callback([this]() {					
+					ImGui::Begin("Hello, world!");
+					ImGui::SetWindowPos(ImVec2(1.0f, 1.0f), ImGuiCond_FirstUseEver);
+					ImGui::Text("%.3f ms/frame", 1000.0f / ImGui::GetIO().Framerate);
+					ImGui::Text("%.1f FPS", ImGui::GetIO().Framerate);
+
+					for (auto& ik : *this->invokeeList)
+					{
+						bool isEnabled = ik->is_enabled();						
+						std::string name = fmt::format("Disable/Enable Invokee [{}]", ik->name());
+						ImGui::Checkbox(name.c_str(), &isEnabled);
+						if (isEnabled != ik->is_enabled())
+						{
+							if (!isEnabled) ik->disable();
+							else ik->enable();
+						}
+					}
+					
+					static std::vector<float> values;
+					values.push_back(1000.0f / ImGui::GetIO().Framerate);
+					if (values.size() > 90) {
+						values.erase(values.begin());
+					}
+					ImGui::PlotLines("ms/frame", values.data(), values.size(), 0, nullptr, 0.0f, FLT_MAX, ImVec2(0.0f, 100.0f));
+					ImGui::End();
+				});
+			}
+		}
+	}
+
+	void update() override
+	{
+		// On Esc pressed,
+		if (gvk::input().key_pressed(gvk::key_code::escape)) {
+			// stop the current composition:
+			gvk::current_composition()->stop();
+		}
+	}
+
+	void render() override
+	{
+		auto mainWnd = gvk::context().main_window();
+
+		// need handling the case where previous invokees had been disabled
+		bool firstInvokeeInChain = !mainWnd->has_consumed_current_image_available_semaphore();
+
+		// Get a command pool to allocate command buffers from:
+		auto& commandPool = gvk::context().get_command_pool_for_single_use_command_buffers(*mQueue);
+
+		// Create a command buffer and render into the *current* swap chain image:
+		auto cmdBfr = commandPool->alloc_command_buffer(vk::CommandBufferUsageFlagBits::eOneTimeSubmit);
+		cmdBfr->begin_recording();
+
+		const auto pushConstants = trianglePart;
+		cmdBfr->handle().pushConstants(mPipeline->layout_handle(), vk::ShaderStageFlagBits::eVertex, 0, sizeof(pushConstants), &pushConstants);
+		
+		// first invokee needs to clear up the backbuffer image: main window render pass clears on load
+		cmdBfr->begin_render_pass_for_framebuffer(firstInvokeeInChain ? gvk::context().main_window()->get_renderpass() : const_referenced(mRenderPass.value()), gvk::context().main_window()->current_backbuffer());
+		cmdBfr->handle().bindPipeline(vk::PipelineBindPoint::eGraphics, mPipeline->handle());
+		cmdBfr->handle().draw(3u, 1u, 0u, 0u);
+		cmdBfr->end_render_pass();
+		cmdBfr->end_recording();
+		
+		if (!firstInvokeeInChain) {
+			mQueue->submit(cmdBfr);
+			mainWnd->handle_lifetime(avk::owned(cmdBfr));
+		}
+		else
+		{
+			auto imageAvailableSemaphore = mainWnd->consume_current_image_available_semaphore();
+			mQueue->submit(cmdBfr, imageAvailableSemaphore);
+			mainWnd->handle_lifetime(avk::owned(cmdBfr));
+		}
+	}
+	
+private: // v== Member variables ==v
+	std::optional<std::vector<invokee*>> invokeeList;
```

Comment:
Members should be named with a prefixed `m`
```suggestion
	std::optional<std::vector<invokee*>> mInvokeeList;
```

Reply:
my mistake - see below.